### PR TITLE
Added some English translations.

### PR DESCRIPTION
**Pull request by TheFerryn**

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+    "help.prefix.description": "Change bot prefix.",
 
     "ping.title": "Ping?",
     "ping.description": ":ping_pong: **%s ms.**",
     "about.title": ":thinking: Who am I?",
     "about.description": "Hello! My name is **Sonata**! I was created by [%s](%s) over a cup of tea and eight hours of hard work at workspace.\n\n> **What I can to do?**\nIt's a good question. I can't do anything yet. :+1:\nMy version: **%s**\nHelp with translate on another language! **[`Click here for redirect to GitHub.`](https://github.com/vlfz/sonata-locales)**",
     "error.title": "%s An error has occurred!",
     "error.description.no_permissions": "Sorry, but you are not allowed to use this command because you don't have this permissions:\n> **%s**",
-    "error.description.no_prefix": "Префикс может быть длиной **от 1 до 8 символов**!",
+    "error.description.no_prefix": "Prefix cannot contain more that 8 characters.",
```
Comment (Mrsasha45op) [inline locales/en.json:27]: more that LMAO

File: locales/en.json [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
     "error.description.no_language": "Bot's language can be only **`%s`**",
     "error.description.no_color": "HEX-code is not provided, or HEX-code was incorrect.\n\n> Maybe you forget type `#` to the beginning of the command.\n> You can pick a color in this website: **[`www.color-hex.com`](https://www.color-hex.com)**",
-    "error.description.cooldown": "У этой команды имеется задержка на использование!\n> Повторите попытку через **%s**",
+    "error.description.cooldown": "This command has a cooldown.!\n> Try again in **%s**",
```
Comment (Mrsasha45op) [inline locales/en.json:30]: a cooldown**.!**

File: locales/en.json (hunk 1 above)
Comment (TheFerryn) [reply locales/en.json:27]: опечатка ладн

File: locales/en.json [hunk 3]
```diff
@@ -27,6 +27,6 @@
     "error.description.no_prefix": "Prefix cannot contain more than 8 characters.",
     "error.description.no_language": "Bot's language can be only **`%s`**",
     "error.description.no_color": "HEX-code is not provided, or HEX-code was incorrect.\n\n> Maybe you forget type `#` to the beginning of the command.\n> You can pick a color in this website: **[`www.color-hex.com`](https://www.color-hex.com)**",
-    "error.description.cooldown": "This command has a cooldown.!\n> Try again in **%s**",
+    "error.description.cooldown": "This command has a cooldown.!\n> Try again in **%s** seconds.",
```
Comment (vlfz) [inline locales/en.json:30]: https://github.com/TheFerryn/sonata-locales/blob/875e6cd9cd12b90a0a86d3b3573ad68135bdcf93/locales/en.json#L13

**Review by vlfz — CHANGES_REQUESTED** [review]

**Review by vlfz — APPROVED** [review]

**Pull request merged**